index.html

```diff
@@ -0,0 +1,44 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li>
+          <a href="#welcome-section">About</a>
+        </li>
+        <li>
+          <a href="#projects">Work</a>
+        </li>
+        <li>
+          <a href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+    
+    <div id="projects">
+      <header class="project-tile">These are some of my projects</header>
```

Comment:
I think it is better to use another selector like {div or section} since you are trying to address here the main content of your page, not the header part. All semantic HTML tags have a relation to the content of the page. Example ✋ 
- best to apply on the header content of the page
- is best to apply on the footer part of the page content.